package.json

```diff
@@ -14,10 +14,10 @@
     "build": "npm run CI && npm run ts && npm run generate-docs"
   },
   "type": "module",
-  "author": "wykhuh",
+  "author": "stevejcl",
   "repository": {
     "type": "git",
-    "url": "https://github.com/DwarfTelescopeUsers/dwarfii_api"
+    "url": "https://github.com/stevejcl/dwarfii_api"
```

Comment:
Change the repo url back to `DwarfTelescopeUsers`

Reply:
no problem